daffodil-core/src/main/scala/edu/illinois/ncsa/daffodil/dsom/RealTermMixin.scala

```diff
@@ -113,17 +113,19 @@ trait RealTermMixin { self: Term =>
             case true => oSeq.possibleNextTerms
             case false => {
 
-              val members = oSeq.groupMembersNoRefs
+              val members = oSeq.groupMembers
 
-              val nextMember =
-                members.dropWhile(m => m != thisTermNoRefs).filterNot(m => m == thisTermNoRefs).headOption
+              val selfAndAfter = members.dropWhile(m => m ne this)
+              val after = selfAndAfter.filterNot(_ eq this)
```

Comment:
Can this be change to this?
```
val after = selfAndAfter.drop(1)
```
``this`` shoudl also be the first element in ``selfAndAfter``, and ``this`` shouldn't appear anywhere else in ``selfAndAfter``. Just a little bit more efficient?